packages/canonical-bridge/src/constants/index.ts

```diff
@@ -0,0 +1,4 @@
+export const MEDIAS = {
```

Comment:
we can use 

```
 ${({ theme }) => theme.mediaQueries.sm} {
   
  }
```
in styled-components, no need to define the media here

Reply:
export const breakpoints = {
  xs: 370,
  sm: 576,
  md: 852,
  lg: 968,
  xl: 1080,
  xxl: 1200,
} as const;

export const mediaQueries = {
  xs: ``,
  sm: `@media screen and (min-width: ${breakpoints.sm}px)`,
  md: `@media screen and (min-width: ${breakpoints.md}px)`,
  lg: `@media screen and (min-width: ${breakpoints.lg}px)`,
  xl: `@media screen and (min-width: ${breakpoints.xl}px)`,
  xxl: `@media screen and (min-width: ${breakpoints.xxl}px)`,
};

the config is here, as a ref